Wikipedia/about.css

```diff
@@ -0,0 +1,33 @@
+function applyDarkThemeLogo() {
+    hideDarkLogo()
+    showLightLogo()
+}
+
+function applyLightThemeLogo() {
+    hideLightLogo()
+    showDarkLogo()
+}
+
+function showDarkLogo() {
+    show(document.getElementById("dark-logo"))
+}
+
+function hideDarkLogo() {
+    hide(document.getElementById("dark-logo"))
+}
+
+function showLightLogo() {
+    show(document.getElementById("light-logo"))
+}
+
+function hideLightLogo() {
+    hide(document.getElementById("light-logo"))
+}
+
+function show(element) {
+    element.style.display = ''
+}
+
+function hide() {
+    element.style.display = 'none'
```

Comment:
Missing `element` parameter here.

Reply:
Thanks. It's the wrong file (js code in .**css** file). Probably, it was added accidentally. I'll remove it.